homework/81352/ts2html/ts2html.flex

```diff
@@ -0,0 +1,106 @@
+%{
+#include <vector>
+#include <iostream>
+#include "token.h"
+#include "highlighter.h"
+std::vector<Token> tokens = std::vector<Token>();
```

Comment:
😢

Reply:
https://github.com/SofiaCPP/IPL/blob/master/demo/flex/lex.yy.c#L563

Reply:
Thanks! It will be updated.